roles/openldap/defaults/main.yml

```diff
@@ -0,0 +1,52 @@
+keystone:
+  ldap:
+    enabled: False 
+
+    # ldap server settings
+    url: ldap://0.0.0.0:389
```

Comment:
This is the url that keystone will used to connect, openldap listens on all interfaces by default.